src/popper/runner_kubernetes.py

```diff
@@ -152,8 +155,8 @@ def _copy_ctx(self):
             _preload_content=False,
         )
 
-        source_file = os.path.join(self._config.workspace_dir, "ctx.tar.gz")
-        destination_file = "/workspace/ctx.tar.gz"
+        source_file = os.path.join(self._config.cache_dir, target_file)
```

Comment:
Can we have the source file at `$POPPER_CACHE_DIR/kubernetes/ctx_{wid}.tar.gz` ? It would help keep the cache directory more organized.

Reply:
Can we bring the `source_file` and `destination_file` variable declarations at the top to just after the `target_file` declaration? In that way, we can prevent joining multiple times.

Reply:
We can also bring the `source_file` and `destination_file` declarations to the top just after the `target_file` variable is declared. In that way, we can prevent joining paths multiple times.

Reply:
Got it! Will do that